docs/Lifecycle/index.md

```diff
@@ -0,0 +1,257 @@
+# **SDL Development Lifecycle**
```

Comment:
@mghiumiusliu,
will you update a [Config.json](https://github.com/smartdevicelink/sdl_markdown_spec#configjson) to show it on the Portal?